platform/podjs/src/manifest-parser/src/manifest.ts

```diff
@@ -5,7 +5,6 @@ import { parse as parseSemVer, SemVer, Range } from "semver";
 
 export interface MainManifest {
```

Comment:
Also, now it's probably not worth the while to keep two different "manifests" separated. The `Main` only has a `name`.